client/src/main/com/sinch/sdk/domains/voice/models/svaml/ActionConnectSip.java

```diff
@@ -89,46 +90,46 @@ public static Builder<?> builder() {
 
   public static class Builder<B extends Builder<B>> {
 
-    DestinationSip destination;
-    Integer maxDuration;
-    String cli;
-    TransportType transport;
-    Boolean suppressCallbacks;
-    Collection<Pair<String, String>> callheaders;
-    MohClassType moh;
+    OptionalValue<DestinationSip> destination = OptionalValue.empty();
+    OptionalValue<Integer> maxDuration = OptionalValue.empty();
+    OptionalValue<String> cli = OptionalValue.empty();
+    OptionalValue<TransportType> transport = OptionalValue.empty();
+    OptionalValue<Boolean> suppressCallbacks = OptionalValue.empty();
+    OptionalValue<Collection<Pair<String, String>>> callheaders = OptionalValue.empty();
+    OptionalValue<MohClassType> moh = OptionalValue.empty();
```

Comment:
Why is the name of this class `MohClassType` and not simply `MohType` like it is with `TransportType` ?

Reply:
This is coming from initial conferenceCalloutRequest schema implementation. OAS is defining a `mohClass` so enums was named at this time. Other schemas using moh are using just a "moh"
You're right naming is not good; and will rename to a more auto documented `MusicOnHoldType` class name

Reply:
Fixed